docker-compose.yaml

```diff
@@ -218,6 +218,8 @@ services:
       - bridge-net
     expose:
       - ${BENTOV2_SERVICE_REGISTRY_INTERNAL_PORT}
+    volumes:
+      - $PWD/lib/service-registry/chord_services.json:/service-registry/bento_service_registry/chord_services.json:rw
```

Comment:
`:rw` grants write access on this file which I think is unnecessary. It would be preferable to not use this for avoiding possible permission issues.